packages/xo-server/src/xo-mixins/xen-servers.mjs

```diff
@@ -91,6 +95,13 @@ export default class {
       }
     })
 
+    this._applianceUuid = Math.random().toString(36).slice(2)
+    app.hooks.on('start', async () => {
+      const value = (await XenStore.read('vm')).trim()
```

Comment:
So this will throw an error every time you start and self-built XO, right?

Reply:
Yes, it will log a warning.